src/main/java/com/amazon/opendistroforelasticsearch/jdbc/protocol/http/JsonCursorQueryRequest.java

```diff
@@ -0,0 +1,39 @@
+package com.amazon.opendistroforelasticsearch.jdbc.protocol.http;
+
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.Parameter;
+import com.amazon.opendistroforelasticsearch.jdbc.protocol.QueryRequest;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class JsonCursorQueryRequest implements QueryRequest  {
```

Comment:
could we reuse the JsonQueryRequest?

Reply:
Both generate different request body.  Not sure how that can be reused.


`JsonQueryRequest` generates this
```
{
  "fetch_size": 0,
  "query": "SELECT ..."
}
```

while `JsonCursorQueryRequest` generates this
```
{
  "cursor":  <cursorId>
}
```